Description: Enable build with older termion versions.
 This patch disables an example build that depend on recent termion versions.
Author: Johann Felix Soden <johfel@debian.org>
Forwarded: not-needed
Last-Update: 2023-07-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/examples/demo/main.rs
+++ b/examples/demo/main.rs
@@ -5,9 +5,7 @@
 mod app;
 #[cfg(feature = "crossterm")]
 mod crossterm;
-#[cfg(all(feature = "termion", not(feature = "crossterm"), not(feature = "termwiz")))]
-mod termion;
-#[cfg(all(feature = "termwiz", not(feature = "crossterm"), not(feature = "termion")))]
+#[cfg(all(feature = "termwiz", not(feature = "crossterm")))]
 mod termwiz;
 
 mod ui;
@@ -28,9 +26,7 @@
     let tick_rate = Duration::from_millis(cli.tick_rate);
     #[cfg(feature = "crossterm")]
     crate::crossterm::run(tick_rate, cli.enhanced_graphics)?;
-    #[cfg(all(feature = "termion", not(feature = "crossterm"), not(feature = "termwiz")))]
-    crate::termion::run(tick_rate, cli.enhanced_graphics)?;
-    #[cfg(all(feature = "termwiz", not(feature = "crossterm"), not(feature = "termion")))]
+    #[cfg(all(feature = "termwiz", not(feature = "crossterm")))]
     crate::termwiz::run(tick_rate, cli.enhanced_graphics)?;
     Ok(())
 }
